### ISSUE TITLE
Pages Out of Date

### ISSUE THREAD

**Issue by wmbeers** [wmbeers/ga-tia#45]
From Karen: Heads up - I've been flooded with concerns this morning about the website. It appears the updates were not made across the board, and some old content has been returned with the new (like the press page on goes up to 2015 now). I'm compiling all the issues now, and will send them as soon as get them all documented.

**Comment by wmbeers** [comment]
Mostly taken care of in cb33f32. I'm leaving this open until Karen finishes a review.

**Comment by wmbeers** [comment]
1. Dan Bodycomb only appears on the bottom of the home page rather than every page. He has replaced Shrujal Amin. 
2. Please update the Fact Sheet (attached).
3. On the SG page, please update '2011 Dollars' to '2017 Dollars.' Sorry, this was my oversight.
4. Facebook feed is not loading in any browser. The handle is @TIAinGeorgia if that helps.
5. Let's go ahead and get the press page up to date. See the second attachment. 2018 will need to be rolled up as previous years, and 2019 can stay unfolded.
6. The 2018 Citizens Review Panel Reports are gone under Program Info & Reports. Please replace with the third attachment, and in line with previous years.

Attachments
1.  [TIA Admin FAQs.pdf](https://github.com/wmbeers/ga-tia/files/3094874/TIA.Admin.FAQs.pdf)
2. [4.18 Press Page Updates.docx](https://github.com/wmbeers/ga-tia/files/3094878/4.18.Press.Page.Updates.docx)
3. [2018 CRP Reports.zip](https://github.com/wmbeers/ga-tia/files/3094880/2018.CRP.Reports.zip)

**Comment by wmbeers** [comment]
Further issues I discovered with News.cshtml:

There are several missing PDFs, such as all of the 2016 Press Releases (among others). They're not in our original source nor in the version of the source Shaun sent me on 3/27. I'm not sure if any were on the server, but the ones I spot checked aren't there.

3.16.16 Warren County TSPLOST.pdf
TIA Story Band1 The Sandersville Progress.pdf
TIA Update Article The Millen News.pdf
Engineering Georgia January_February 2016.pdf
06.22.16 _Columbus METRA Buses.pdf
06.15.16 - TIA Discretionary Funds.pdf
05.16.16 _ Jones Bridge Road Release.pdf
03.14.16 Custer RD Release.pdf
02 11 16_TIA Band 1 Success.pdf

(That's not a comprehensive list of what's missing, just what I've found so far in spot-checking.)

Shaun checked, and none of these are in backups of current prod, so this is a long-standing issue.
